custom_components/zaptec/api.py

```diff
@@ -323,9 +323,12 @@ async def _stream(self, cb=None, ssl_context=None):
 
             self._stream_receiver = None
             async with servicebus_client:
-                receiver = servicebus_client.get_subscription_receiver(
-                    topic_name=conf["Topic"], subscription_name=conf["Subscription"]
-                )
+                async with servicebus_client:
+                    receiver = await hass.async_add_executor_job(
```

Comment:
I did not test this properly. hass is not defined at here. Not sure if this change actually fixes what I thought it did since it seemed to work first time I tried. I can send a restart charger command but not a resume command. 

What will the best way be to get hass here?